### PR TITLE
Tickets/dm 24671

### PR DESCRIPTION
**Pull request by couger01**

### CODE REVIEW

File: python/lsst/ts/ATHexapod/gcserror.py [hunk 1]
```diff
@@ -32,453 +32,229 @@
 class PIError(enum.Enum):
     """Provide enum for PI Errors."""
     E_1_COM_ERROR = -1
```
Comment (tribeiro) [inline python/lsst/ts/ATHexapod/gcserror.py:34]: Why are you removing the documentation strings for this file?

File: Jenkinsfile.conda [hunk 2]
```diff
@@ -3,7 +3,7 @@ pipeline {
     environment {
         package_name = "athexapod"
         dockerImageName = "lsstts/conda_package_builder:latest"
-        container_name = "salobj_${BUILD_ID}_${JENKINS_NODE_COOKIE}"
+        container_name = "athexapod_conda_${BUILD_ID}_${JENKINS_NODE_COOKIE}"
```
Comment (tribeiro) [inline Jenkinsfile.conda]: To keep the Jenkinsfile generic, I would recommend not using the package name on places like this. I would just use 

```
container_name = "conda_${BUILD_ID}_${JENKINS_NODE_COOKIE}"
```

File: Jenkinsfile.conda [hunk 3]
```diff
@@ -37,7 +37,10 @@ pipeline {
                 }
             }
         }
-        stage("Create ATDome Conda package") {
+        stage("Create ATHexapod Conda package") {
```
Comment (tribeiro) [inline Jenkinsfile.conda]: For generality I would recommend just 

```
Create Conda package
```

File: Jenkinsfile.conda [hunk 4]
```diff
@@ -46,7 +49,25 @@ pipeline {
                 }
             }
         }
-        stage("Push ATDome Conda package") {
+        stage("Create ATHexapod Conda Dev package") {
```
Comment (tribeiro) [inline Jenkinsfile.conda]:
```
Create Conda Dev package
```

File: Jenkinsfile.conda [hunk 5]
```diff
@@ -46,7 +49,25 @@ pipeline {
...
+            steps {
+                script {
+                    sh """
+                        docker exec ${container_name} sh -c 'cd ~/source/conda && source ~/miniconda3/bin/activate && source "\$OSPL_HOME/release.com" && conda build -c lsstts/label/dev --prefix-length 100 .'
+                    """
+                }
+            }
+        }
+
+        stage("Push ATHexapod Conda Release package") {
```
Comment (tribeiro) [inline Jenkinsfile.conda]:
```
Push Conda Release package
```

**Review by tribeiro — CHANGES_REQUESTED** [review]
Is there a good reason for removing the error codes docstring in `python/lsst/ts/ATHexapod/gcserror.py`? I think before removing the text here we may want to move them to a documentation page. Or, maybe even better, move them to the class docstring so it is rendered by the doc package.

I guess there's no need to do that in this ticket, as long as you restore the documentation string. You can make a `#TODO` comment and create a ticket for that to be done later.

File: python/lsst/ts/ATHexapod/gcserror.py (hunk 1 above)
Comment (couger01) [reply python/lsst/ts/ATHexapod/gcserror.py:34]: Created ticket.

**Review by tribeiro — APPROVED** [review]

**Pull request merged**